Apps/EGIAmpServer/CMakeLists.txt

```diff
@@ -39,10 +41,16 @@ target_link_libraries(${PROJECT_NAME}
 	Qt5::Widgets
 	LSL::lsl
 	Boost::system
+	Boost::date_time
+	Boost::regex
 	Threads::Threads
 )
 
+
 target_compile_features(${PROJECT_NAME} PRIVATE cxx_auto_type)
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
I'd rather have it as target_compile_feature since those can be upgraded but not overwritten

Reply:
Done.